app/controllers/dictionary_controller.rb

```diff
@@ -0,0 +1,15 @@
+require 'json'
+
+class DictionaryController < ApplicationController
```

Comment:
I prefer this would be more restful maybe nested within the dictionary directory and create being one of the 7 restful actions

class Dictionary::SearchController
    def create
    end
end

Reply:
Check out this article http://jeromedalbert.com/how-dhh-organizes-his-rails-controllers/